composer.json

```diff
@@ -34,8 +34,11 @@
     },
     "bin": ["bin/generate_schema_objects"],
     "require": {
-        "php": "^7.1",
-        "gmostafa/php-graphql-client": "^1.6"
+        "php": "^7.1 || ^8.0",
+        "gmostafa/php-graphql-client": "^1.6 || dev-master"
```

Comment:
Please update this to point to v1.9 instead of dev-master

Reply:
Done!